solidity/random-beacon/test/fixtures/index.ts

```diff
@@ -27,16 +27,16 @@ export const dkgState = {
 }
 
 export const params = {
-  relayRequestFee: 0,
+  relayRequestFee: to1e18(100),
   relayEntrySubmissionEligibilityDelay: 10,
-  relayEntryHardTimeout: 5760,
+  relayEntryHardTimeout: 100,
```

Comment:
Do the parameters' values affect tests duration? I'd love to keep the values as close as possible to the actually used in the mainnet contracts deployement.

Reply:
Yep, as I mentioned in the description, the higher value of such a param, the higher `mineBlocks` execution time. This is especially visible in scenarios related to relay entry timeout. This can't be easily optimized with snapshotting because each `mineBlocks` must be invoked at least one time.